test/grappler/ssd_mobilenet_v1.patch

```diff
@@ -0,0 +1,167 @@
++    results.dataset['categories'] = self.dataset['categories']
+     if self._detection_type == 'bbox':
+       for idx, ann in enumerate(annotations):
+         bb = ann['bbox']
+diff --git a/research/object_detection/model_main.py b/research/object_detection/model_main.py
+index 2082c84..6bce629 100644
+--- a/research/object_detection/model_main.py
++++ b/research/object_detection/model_main.py
+@@ -25,6 +25,9 @@ import tensorflow as tf
+ from object_detection import model_hparams
+ from object_detection import model_lib
+ 
++# Log output to console
++tf.logging.set_verbosity(tf.logging.INFO)
++
+ flags.DEFINE_string(
+     'model_dir', None, 'Path to output model directory '
+     'where event and checkpoint files will be written.')
+diff --git a/research/object_detection/samples/configs/ssd_mobilenet_v1_300x300_coco14_sync.config b/research/object_detection/samples/configs/ssd_mobilenet_v1_300x300_coco14_sync.config
+index c21a1fa..0476942 100644
+--- a/research/object_detection/samples/configs/ssd_mobilenet_v1_300x300_coco14_sync.config
++++ b/research/object_detection/samples/configs/ssd_mobilenet_v1_300x300_coco14_sync.config
+@@ -142,12 +142,12 @@ model {
+ }
+ 
+ train_config: {
+-  fine_tune_checkpoint: "PATH_TO_BE_CONFIGURED/model.ckpt"
+-  batch_size: 1024
++  fine_tune_checkpoint: "/localdisk/dataset/pretrained_models/mobilenet_v1/mobilenet_v1_1.0_224.ckpt"
++  batch_size: 64
+   sync_replicas: true
+   startup_delay_steps: 0
+   replicas_to_aggregate: 8
+-  num_steps: 20000
++  num_steps: 200000
+   data_augmentation_options {
+     random_horizontal_flip {
+     }
+@@ -161,7 +161,7 @@ train_config: {
+       learning_rate: {
+         cosine_decay_learning_rate {
+           learning_rate_base: 0.9
+-          total_steps: 20000
++          total_steps: 200000
+           warmup_learning_rate: 0.3
+           warmup_steps: 1000
+         }
+@@ -176,22 +176,23 @@ train_config: {
+ 
+ train_input_reader: {
+   tf_record_input_reader {
+-    input_path: "PATH_TO_BE_CONFIGURED/mscoco_train.record-00000-of-00100"
++    input_path: "/localdisk/dataset/mscoco2014/tf_records_train/coco_train.record-00000-of-00100"
+   }
+-  label_map_path: "PATH_TO_BE_CONFIGURED/mscoco_label_map.pbtxt"
++  label_map_path: "/localdisk/sairamsu/tf_training/models/research/object_detection/data/mscoco_label_map.pbtxt"
+ }
+ 
+ eval_config: {
+   metrics_set: "coco_detection_metrics"
+   use_moving_averages: false
++  max_evals: 1
+   num_examples: 8000
+ }
+ 
+ eval_input_reader: {
+   tf_record_input_reader {
+-    input_path: "PATH_TO_BE_CONFIGURED/mscoco_val.record-00000-of-00010"
++    input_path: "/localdisk/dataset/mscoco2014/tf_records_val/coco_val.record"
+   }
+-  label_map_path: "PATH_TO_BE_CONFIGURED/mscoco_label_map.pbtxt"
++  label_map_path: "/localdisk/sairamsu/tf_training/models/research/object_detection/data/mscoco_label_map.pbtxt"
+   shuffle: false
+   num_readers: 1
```

Comment:
these changes in the config file are not required for grappler and can be removed from the patch.

Reply:
Not sure how these got added, will remove them.